pkg/crc/machine/start.go

```diff
@@ -443,7 +443,11 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 
 	logging.Info("Starting OpenShift cluster... [waiting for the cluster to stabilize]")
 
-	if err := cluster.WaitForClusterStable(ctx, ocConfig); err != nil {
+	lister, err := lister(instanceIP, crcBundleMetadata)
```

Comment:
wondering if `kubeclient` or such would be a better name than `lister`? I'm kind of confused that you create something called `lister` here, and then have to call `lister.ConfigV1().ClusterOperators()` in order to get a `lister` which is usable by `cluster.WaitForClusterStable`.
Maybe `lister` should live in the `cluster` package with just `cluster.WaitForClusterStable(ctx, instanceIP, crcBundleMetadata)`?

Reply:
yes done